djmoney/models/fields.py

```diff
@@ -286,7 +286,7 @@ def formfield(self, **kwargs):
         return super(MoneyField, self).formfield(**defaults)
 
     def value_to_string(self, obj):
-        if VERSION == (1, 8):
+        if VERSION[:2] == (1, 8):
```

Comment:
Small tip, not necessary: But you can write this a bit like you would with pip: `Django>=1.8,<1.9`:

```
>>> (1, 8, 0) >= (1, 8) < (1, 9, 0)
True
```

Reply:
So would be: `(1, 8) >= VERSION < (1,9)`